resolwe_bio/processes/reports/amplicon_multi_report.yml

```diff
@@ -89,6 +79,11 @@
       mkdir multisample_report
       # Bash syntax for computing Array size is clashing with jinja comments syntax.
       # With "raw" we tell jinja to leave this line alone.
+      {% set template = 'multi-sample-template'|data_by_slug %}
+      {% set logo = 'genialis-logo'|data_by_slug %}
+      TEMPLATE='/data_all/{{ template | int }}/multireport-template.tex'
```

Comment:
@mstajdohar @JureZmrzlikar Could we bundle the logo & template as a "resource" to a script in the docker image somehow?

Reply:
We talked a bit about this with @dblenkus . He suggests thatwe modify frontend call to this process in way that is is able to accomodate other inputs or get them from configuration. But i guess this will take time...

Reply:
> He suggests thatwe modify frontend call to this process in way that is is able to accomodate other inputs or get them from configuration.

This would take too long. We either put this in Docker or write a DSS process that calls this and inserts the logo and template in the inputs. The DSS process would get the logo and template by slug. I do not believe there is and advantage of writing a DSS process over having the logo and template in the Docker.

Reply:
Putting the template into the Docker image is preferred solution, if there is only one template.